src/ResourceManager/Automation/Commands.Automation.Test/AutomationTests.cs

```diff
@@ -79,5 +79,29 @@ public void TestAutomationRunbookWithParameter()
         {
             RunPowerShellTest("Test-RunbookWithParameter -runbookPath ScenarioTests\\Resources\\fastJob.ps1  @{'nums'='[1,2,3,4,5,6,7]'}  28");
         }
+
+        [Fact]
+        [Trait(Category.Service, Category.Automation)]
+        [Trait(Category.RunType, Category.LiveOnly)]
```

Comment:
Why can these tests not be run as Acceptance tests?

Reply:
Yes it can run. I have added
        [Trait(Category.AcceptanceType, Category.CheckIn)]
        [Trait(Category.AcceptanceType, Category.BVT)]

Reply:
I don't see that change here